### ISSUE TITLE
PVM build for AWS failing

### ISSUE THREAD

**Issue by netrounds-jonas** [andsens/bootstrap-vz#388]
## Error
```python
Traceback (most recent call last):
  File "./cloud-images/bootstrap-vz/bootstrap-vz", line 5, in <module>
    main()
  File "/home/ubuntu/proberepo/cloud-images/bootstrap-vz/bootstrapvz/base/main.py", line 29, in main
    dry_run=opts['--dry-run'])
  File "/home/ubuntu/proberepo/cloud-images/bootstrap-vz/bootstrapvz/base/main.py", line 111, in run
    tasklist.run(info=bootstrap_info, dry_run=dry_run)
  File "/home/ubuntu/proberepo/cloud-images/bootstrap-vz/bootstrapvz/base/tasklist.py", line 44, in run
    task.run(info)
  File "/home/ubuntu/proberepo/cloud-images/bootstrap-vz/bootstrapvz/common/tasks/filesystem.py", line 119, in run
    filter(is_additional, partitions),
  File "/home/ubuntu/proberepo/cloud-images/bootstrap-vz/bootstrapvz/common/tasks/filesystem.py", line 114, in is_additional
    partition.name not in ["boot", "swap", "root"])
AttributeError: 'SinglePartition' object has no attribute 'name'
```

## Reproducing the error
Build the following YAML using bootstrap-vz `$ bootstrap-vz test-pvm.yml` and replace the `${VARIABLES}` with the appropriate value.
```yaml
---
name: debian-{system.release}-{system.architecture}-{provider.virtualization}-{%Y}
-{%m}-{%d}-ebs
provider:
  name: ec2
  virtualization: pvm
  bucket: ${AWS_S3_BUCKET}
  region: ${AWS_EC2_REGION}
  credentials:
    access-key: ${AWS_ACCESS_KEY}
    secret-key: ${AWS_SECRET_KEY}
  description: Debian {system.release} {system.architecture}
bootstrapper:
  workspace: /target
system:
  release: wheezy
  architecture: i386
  bootloader: pvgrub
  charmap: UTF-8
  locale: en_US
  timezone: UTC
volume:
  backing: ebs
  partitions:
    type: none
    root:
      filesystem: ext4
      size: 8GiB
packages:
  mirror: http://cloudfront.debian.net/debian
plugins:
  cloud_init:
    metadata_sources: Ec2
    username: admin
```
## Possible fix
It seems to me that the parsing of
```yaml
partitions:
  type: none
  root:
    filesystem: ext4
    size: 8GiB
```
does not recognise it as an unformatted partition. A corresponding YAML for HVM builds and deploys fine.

**Comment by alexfrancavilla** [comment]
I'm having the exact same issue with the official wheezy HVM manifest on an EC2 Debian 8 instance.

**Comment by andsens** [comment]
10-4. I'll try to get a look at it in the weekend.

**Comment by andsens** [comment]
Fixed in 19e15d15bfb3f886e80ff38dd7abd9976b261221 :-)
